javascript-demo/variables.js

```diff
@@ -0,0 +1,27 @@
+var a;
+a = 5;
+
+let b;
+b = 4;
+
+const c = 10;
+
+var d = 5;
+
+let e = 4;
+
+const x = "x", y = "y", z = "z";
+
+let ab, bc, ac;
+ab = bc = ac = "abc";
+
+const aa = "aabbcc";
```

Comment:
In JavaScript, we should use single quotes `''` for strings. Refer this [article](https://github.com/airbnb/javascript?tab=readme-ov-file#strings--quotes)